di-ipv-credential-issuer-stub/src/main/resources/data/criStubEvidencePayloads.json

```diff
@@ -408,6 +408,58 @@
         ]
       }
     },
+    {
+      "criType": "DWP KBV (Stub)",
+      "label": "Passed DWP KBV (Stub)",
+      "payload": {
+        "type": "IdentityCheck",
+        "txn": "8KJTLGN7QX",
+        "verificationScore": 2,
+        "checkDetails": [
+          {
+            "kbvResponseMode": "free_text",
+            "kbvQuality": 3,
+            "checkMethod": "kbv"
+          },
+          {
+            "kbvResponseMode": "multiple_choice",
+            "kbvQuality": 3,
+            "checkMethod": "kbv"
+          },
+          {
+            "kbvResponseMode": "multiple_choice",
+            "kbvQuality": 2,
+            "checkMethod": "kbv"
+          }
+        ]
+      }
+    },
+    {
+      "criType": "DWP KBV (Stub)",
+      "label": "Failed DWP KBV (Stub)",
+      "payload": {
+        "type": "IdentityCheck",
+        "txn": "8KJTLGN7QX",
+        "verificationScore": 0,
+        "ci": ["V03"],
+        "failedCheckDetails": [{
+					"kbvResponseMode": "free_text",
```

Comment:
v minor but could we reduce the indent

Reply:
In the editor this is fine, no crazy indentation, I think its just GitHub displaying it weirdly